assets/javascripts/modules/checkout/checkout.js

```diff
@@ -135,13 +138,25 @@ define(['$',
     };
 
     var finishAccount = function () {
-        if ($FINISH_ACCOUNT_SUBMIT.length) {
+        if ($FINISH_ACCOUNT_FORM.length) {
             bean.on($FINISH_ACCOUNT_SUBMIT[0], 'click', function (e) {
                 e.preventDefault();
 
                 if (validations.validateFinishAccount()) {
-                    bean.off($FINISH_ACCOUNT_SUBMIT[0], 'click');
-                    bean.fire($FINISH_ACCOUNT_SUBMIT[0], 'click');
+                    var action = $FINISH_ACCOUNT_FORM.attr('action');
+                    var formData = ajax.reqwest.serialize($FINISH_ACCOUNT_FORM[0]);
+
+                    ajax({
+                        url: action,
+                        method: 'POST',
+                        data: formData
+                    }).then(function () {
```

Comment:
Do we not need to check anything about the response here to make sure the registration was actually successful?